src/ReactiveUI/Interactions.cs

```diff
@@ -200,14 +200,16 @@ public IDisposable RegisterHandler(Func<InteractionContext<TInput, TOutput>, Tas
         /// <returns>
         /// A disposable which, when disposed, will unregister the handler.
         /// </returns>
-        public IDisposable RegisterHandler(Func<InteractionContext<TInput, TOutput>, IObservable<Unit>> handler)
+        public IDisposable RegisterHandler<TDontCare>(Func<InteractionContext<TInput, TOutput>, IObservable<TDontCare>> handler)
```

Comment:
Should we keep `public IDisposable RegisterHandler(Func<InteractionContext<TInput, TOutput>, IObservable<Unit>> handler)` as an alias method to the new signature but annotate it with `[Obsolete("Please use RegisterHandler<T> instead?")]` as a migration path? I want to stop making hard public API contract changes. Your thoughts?